components/contribution-flow/StepDetails.js

```diff
@@ -124,6 +124,14 @@ const StepDetails = ({ onChange, stepDetails, collective, tier, showPlatformTip,
 
       {!isFixedContribution ? (
         <Box mb="30px">
+          {amount && amount < minAmount && (
```

Comment:
We need the check to be a boolean, otherwise we get an unexpected `0`:

![image](https://github.com/user-attachments/assets/6f5c879d-ece4-495a-9198-e81f53b64c80)

But actually, even if it's `0`, we want to show the error message. So I'd suggest:

```suggestion
          {typeof amount === 'number' && !isNaN(amount) && amount < minAmount && (
```